src/scripts/oakitems/OakItem.ts

```diff
@@ -31,7 +31,7 @@ class OakItem {
         if (OakItemRunner.selectedItem().canUpgrade()) {
             return "Cost: " + GameConstants.OAKITEM_MONEY_COST[OakItemRunner.selectedItem().level()];
         }
-        return OakItemRunner.selectedItem().getNormalizedPlayerExp() + "/" + OakItemRunner.selectedItem().getNormalizedRequiredExp();
+        return OakItemRunner.inspectedItem().getNormalizedPlayerExp() + "/" + OakItemRunner.inspectedItem().getNormalizedRequiredExp();
```

Comment:
This line gave me so much trouble, I couldn't figure out why it wasn't updating the value to the item that was being hovered over.. didn't expect it to be using `selectedItem` as nothing else was :laughing: